src/storage/src/client/upload_object/unbuffered.rs

```diff
@@ -325,37 +267,4 @@ mod tests {
         }
         Ok(())
     }
-
-    #[test_case("projects/p", "projects%2Fp")]
-    #[test_case("kebab-case", "kebab-case")]
-    #[test_case("dot.name", "dot.name")]
-    #[test_case("under_score", "under_score")]
-    #[test_case("tilde~123", "tilde~123")]
-    #[test_case("exclamation!point!", "exclamation%21point%21")]
-    #[test_case("spaces   spaces", "spaces%20%20%20spaces")]
-    #[test_case("preserve%percent%21", "preserve%percent%21")]
-    #[test_case(
-        "testall !#$&'()*+,/:;=?@[]",
-        "testall%20%21%23%24%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D"
-    )]
-    #[tokio::test]
-    async fn test_percent_encoding_object_name(name: &str, want: &str) -> Result {
```

Comment:
why is this removed?

Reply:
Because it duplicates the exact same testing done in `upload_object.rs`.  I think this test really belongs in `client.rs`.